src/main/java/com/github/edouardswiac/zerotier/ZTService.java

```diff
@@ -25,4 +30,17 @@
   ZTNetworkMember getNetworkMember(String networkId, String address);
   void updateNetworkMember(ZTNetworkMember networkMember);
   void deleteNetworkMember(String networkId, String address);
+
+  List<ZTNet> getNets();
```

Comment:
Agreed.
Only one note: API-wise my `ZTNet` is exactly the same as Your `ZTNetwork`, I merely added some more fields.
IMHO only one of them should survive.
Your choice ;)